### ISSUE TITLE
Download page suggests that stable version is not mature enough

### ISSUE THREAD

**Issue by milosz** [antirez/redis-io#147]
Other versions -> Old (3.2) contains text "while 4.0 matures more in the next months".
Is this still true? It was  committed on 14 Jul 2017 (07c80140d4192870eeadfc7be5e7b2cf0e47a070) in download.json file. It suggests to stick with 3.2 instead of current stable.

**Comment by itamarhaber** [comment]
Hello @milosz 

Thanks for noticing. No, this isn't correct. 4 is stable enough, the doc needs to be updated - would you care to make the PR?

**Comment by milosz** [comment]
Hello,

I have created PR https://github.com/antirez/redis-io/pull/148

Thanks,
Milosz

**Comment by milosz** [comment]
Hi, Do you need anything else from me?

**Comment by itamarhaber** [comment]
No, thanks. It is waiting for a maintainer to be reviewed & merged (cc @antirez)

**Comment by milosz** [comment]
Ok. Thanks.
